prime-dotnet-webapi/Models/BaseAuditable.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+    public class BaseAuditable : IAuditable
```

Comment:
Where do we set these values?

Reply:
These get set by the ApiDbContext as part of the SaveChanges and SaveChangesAsync methods.